djstripe/fields.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **defaults)
 
 
-class StripeQuantumCurrencyAmountField(models.IntegerField):
+class StripeQuantumCurrencyAmountField(models.BigIntegerField):
     """
     A field used to store currency amounts in cents (etc) as per stripe.
```

Comment:
can you update here to mention why (same docstring as per the test would be fine).